config/config.defaults.yml

```diff
@@ -253,6 +253,18 @@ default: &default_settings
   # Max filesize for uploaded image (bytes)
   max_image_filesize: 104857600
 
+  # Memory limit for image processing - passed to ImageMagick -limit memory (MAGICK_MEMORY_LIMIT)
+  # See http://www.imagemagick.org/script/resources.php
+  # Example value:
+  # image_processing_memory_limit: 512MiB
+  image_processing_memory_limit:
```

Comment:
I had trouble deciding what the "sensible" defaults would be so left the default behaviour unchanged. What do you think?